src/components/Task.tsx

```diff
@@ -1,8 +1,20 @@
 import React from "react"
 
-export default function Task({ task, completeTask, removeTask }) {
-  const checkMark = task.completed
-    ? "m8 13 2.165 2.165a1 1 0 0 0 1.521-.126L16 9"
+interface Task {
+  title: string,
+  completed: boolean,
+  key: string
+}
+
+type Props = {
+  task: Task,
+  completeTask: Function,
+  removeTask: Function
+}
```

Comment:
Same treatment as above:
```suggestion
interface TaskProps {
  task: Task;
  completeTask: (task: Task) => void;
  removeTask: (key: string) => void;
}
```